python/tests/engine/test_python.py

```diff
@@ -1741,22 +1735,19 @@ def test_convert_to_unix_timestamp_str(self, mocker):
             "hsfs.util.get_timestamp_from_date_string"
         )
 
-        python_engine = python.Engine()
-
         mock_util_get_timestamp_from_date_string.return_value = 1483225200000
 
         # Act
-        result = python_engine._convert_to_unix_timestamp(t="2017-01-01 00-00-00-000")
+        result = util.convert_event_time_to_timestamp(
+            event_time="2017-01-01 00-00-00-000"
+        )
 
         # Assert
         assert result == 1483225200000
 
     def test_convert_to_unix_timestamp_int(self):
-        # Arrange
-        python_engine = python.Engine()
-
         # Act
-        result = python_engine._convert_to_unix_timestamp(t=1483225200)
+        result = util.convert_event_time_to_timestamp(event_time=1483225200)
```

Comment:
I am pro a new test class `test_utils.py` where the new `util.convert_event_time_to_timestamp` gets tested with various inputs. The tests should cover all input types (int, string, timestamp, datetime.datetime, datetime.date), and in particular different supported and not supported string-based input formats, timestamps and datetimes with timezones, and other possible edge cases.

Reply:
added couple of more tests in `test_python.py`